src/lib.rs

```diff
@@ -879,12 +906,34 @@ impl<T: Counter> Histogram<T> {
     /// from the resulting histogram without warning. Since the values are clamped, the histogram
     /// will also not be resized to accomodate the value, even if auto-resize is enabled.
     pub fn saturating_record_n(&mut self, value: u64, count: T) {
-        self.record_n_inner(value, count, true).unwrap()
+        self.record_n_inner(false, value, count, true).unwrap()
+    }
+
+    /// Unrecord multiple samples for a value in the histogram, removing from the value's current count.
+    ///
+    /// `count` is the number of occurrences of this value to record.
+    ///
+    /// This method cannot fail, as values that are too small or too large to be recorded will
+    /// automatically be clamed to be in range. Be aware that this *will* hide extreme outliers
+    /// from the resulting histogram without warning. Since the values are clamped, the histogram
+    /// will also not be resized to accomodate the value, even if auto-resize is enabled.
+    pub fn saturating_unrecord_n(&mut self, value: u64, count: T) {
+        self.record_n_inner(true, value, count, true).unwrap()
     }
 
-    fn record_n_inner(&mut self, mut value: u64, count: T, clamp: bool) -> Result<(), RecordError> {
+    fn record_n_inner(
+        &mut self,
+        sub: bool,
```

Comment:
I think I'd prefer that we add
```rust
enum Op {
  Add,
  Sub,
}
```

as it will make the callsites a lot easier to vet. For example, it'll read:
```rust
self.record_n_inner(Op::Add, value, count, true)
```
instead of
```rust
self.record_n_inner(false, value, count, true)
```